res/values/strings.xml

```diff
@@ -465,6 +465,28 @@
     <!-- the explanation is shown (1) as a modal dialog with the buttons "Cancel" and "Continue" as well as (2) as a device message -->
     <string name="aeap_explanation">You changed your email address from %1$s to %2$s.\n\nIf you now send a message to a verified group, contacts there will automatically replace the old with your new address.\n\nIt\'s highly advised to set up your old email provider to forward all emails to your new email address. Otherwise you might miss messages of contacts who did not get your new address yet.</string>
 
+
+    <!-- Multi Device -->
+    <string name="multidevice_title">Add Another Device</string>
```

Comment:
"Add Second Device" may also be a handy name, even if it is little less fitting in case of a third device.

"Link Device" was sorted out somehow, as this implies a bit that one device depends on the other, as of whatsapp or signal.

Reply:
I think it is fine as it is, but if you are looking to change it, "Add Second Device"  and "Add New Device" look fine also

Reply:
What about `Copy to new device`?

Reply:
"Copy to new device" sounds nice at a first glance, however, somehow i had the feeling it makes things less clear and less  catchy.

thinking a bit longer about it, i think, it is about the changed subject - "Add Device" has the subject "Device" - "Copy to new Device" has only an implicit subject.

Reply:
finally, after some one-to-one conversations, decided to go for "Second Device" at https://github.com/deltachat/deltachat-android/pull/2493/commits/5f271d8d036361f2593952e91944ba404385bd6c , this is a but catchier and  was also the intuitive wording used in core and on desktop first; let's give it a try, i do not think it make huge differences :)